src/cli/cli-definitions.js

```diff
@@ -174,5 +174,15 @@ export default {
   "verbose": {
     env: "VERBOSE",
     help: "Set the logging to verbose"
+  },
+  "logLevel": {
+    help: "Sets the log level"
+  },
+  "enableConfigChanges": {
+    env: "ENABLE_CONFIG_CHANGES",
```

Comment:
Ah good point. Lets leave the setting and have it default to enabled then, and also add to the docs that it exists primarily for optimization purposes. Parse philosophy is typically to prioritize better user experience over most other thing, and I think having the dashboard work out of the box for changing settings is better user experience. Also I think given that optimization is the purpose of this setting, we don't need the env var.